src/NServiceBus.Core/Licensing/LicenseReminder.cs

```diff
@@ -40,6 +42,23 @@ protected internal override void Setup(FeatureConfigurationContext context)
             }
         }
 
+        static object GenerateLicenseDiagnostics(LicenseManager licenseManager)
+        {
+            return new
+            {
+                licenseManager.result.License.RegisteredTo,
+                licenseManager.result.License.LicenseType,
+                licenseManager.result.License.Edition,
```

Comment:
One of the things that came up when we changed the license files is that customers may look inside of them and see Edition but not Tier. The new licensing model does not have Editions, only Tiers. As it was considered that this may be a source of confusion, we decided to add Tier into the license file (even though it is unused). Should we consider adding it here and just repeating the Edition (which is what the license file does)?

Reply:
Should I just rename the Edition attribute to "Tier" instead of repeating it?

Reply:
No, because some (older) licenses still have an Edition.